libs/libicu/Makefile

```diff
@@ -0,0 +1,133 @@
+#
+# Copyright (C) 2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=icu
+PKG_VERSION:=54.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=icu4c-54_1-src.tgz
+PKG_SOURCE_URL:=http://download.icu-project.org/files/icu4c/$(PKG_VERSION)
```

Comment:
Martin, thanks for the hint.


On 24 March 2015 at 21:09, Martin Hasoň <notifications@github.com> wrote:

> In libs/libicu/Makefile
> <https://github.com/openwrt/packages/pull/1050#discussion_r27073386>:
>
> > @@ -0,0 +1,133 @@
> > +#
> > +# Copyright (C) 2015 OpenWrt.org
> > +#
> > +# This is free software, licensed under the GNU General Public License v2.
> > +# See /LICENSE for more information.
> > +#
> > +
> > +include $(TOPDIR)/rules.mk
> > +
> > +PKG_NAME:=icu
> > +PKG_VERSION:=54.1
> > +PKG_RELEASE:=1
> > +
> > +PKG_SOURCE:=icu4c-54_1-src.tgz
> > +PKG_SOURCE_URL:=http://download.icu-project.org/files/icu4c/$(PKG_VERSION)
>
> Typo PKG_SOURCE_URL:=
> http://download.icu-project.org/files/icu4c/$(PKG_SOURCE)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openwrt/packages/pull/1050/files#r27073386>.
>



-- 

Carlos Miguel Ferreira
Researcher at Telecommunications Institute
Aveiro - Portugal
Work E-mail - cmf@av.it.pt
Skype & GTalk -> carlosmf.pt@gmail.com
LinkedIn -> http://www.linkedin.com/in/carlosmferreira